lib/matplotlib/image.py

```diff
@@ -422,6 +441,10 @@ def _make_image(self, A, in_bbox, out_bbox, clip_bbox, magnification=1.0,
                 A_scaled += 0.1
                 A_resampled = np.zeros((out_height, out_width),
                                        dtype=A_scaled.dtype)
+                if self.get_antialiased() and A.shape[0] > out_height:
+                    self.set_interpolation('lanczos')
```

Comment:
can we use a simpler filter?

Reply:
I don't think lanczos is very complicated.  https://en.wikipedia.org/wiki/Lanczos_resampling

The other filters (i.e. Gaussian) don't have a radius associated with them, for some reason I can't fathom, so I didn't really play with them, but will do...

Reply:
Simpler filter seems to work great.  See update above: